src/io/io.cpp

```diff
@@ -0,0 +1,63 @@
+//
+// Created by Sameer Raj on 02/08/24.
+//
+#include "../../headers/io/io.h"
+#include "../../headers/configs/constants.h"
+#include <fstream>
+
+#include <string>
+
```

Comment:
Suggestion: MMap files might be useful here, seekg, seekp all go through user mode to kernel mode again and again causing performance penalty, mmap files cause 1 single system call and then from then on its in memory